pandas/core/frame.py

```diff
@@ -4872,12 +4874,15 @@ def apply(self, func, axis=0, broadcast=None, raw=False, reduce=None,
             of those. However if the apply function returns a Series these
             are expanded to columns.
 
-            .. versionadded:: 0.23.0
+            .. versionadded:: 0.23.0.
 
         args : tuple
             Positional arguments to pass to function in addition to the
-            array/series
-        Additional keyword arguments will be passed as keywords to the function
+            array/series.
+
+        kwds : dictionary
```

Comment:
@jorisvandenbossche remind me, what's the policy here? Just `kwargs`, with no type?

Reply:
Though probably just `kwds` in this case.

Reply:
`**kwargs` with no type